invenio_rdm_records/assets/semantic-ui/js/invenio_rdm_records/src/deposit/fields/CreatibutorsField/CreatibutorsModal.js

```diff
@@ -679,16 +680,18 @@ export class CreatibutorsModal extends Component {
                     (showPersonForm &&
                       _get(values, typeFieldPath) === CREATIBUTOR_TYPE.PERSON)) && (
                     <div>
-                      <SelectField
-                        fieldPath={roleFieldPath}
-                        label={i18next.t("Role")}
-                        options={roleOptions}
-                        placeholder={i18next.t("Select role")}
-                        {...(this.isCreator() && { clearable: true })}
-                        required={!this.isCreator()}
-                        optimized
-                        scrolling
-                      />
+                      <Overridable id="InvenioRdmRecords.Deposit.CreatibutorsModalRoleSelectField.Container">
```

Comment:
you are missing passing props to this component. Without props passed to overridable you will not be able to access any props in your overridden cmp (fe. you don't want to override fieldPath, because the form will not work)

Reply:
@mkloeppe Could you look into this?